systemd-service/storageos.service

```diff
@@ -11,8 +11,8 @@ TimeoutStartSec=0
 EnvironmentFile=/etc/default/storageos
 ExecStartPre=-/usr/bin/docker kill storageos
 ExecStartPre=-/usr/bin/docker rm storageos
-ExecStartPre=/usr/bin/docker pull storageos/node:1.5.3
-ExecStart=/usr/bin/docker run --name storageos -e DISABLE_TELEMETRY=true -e HOSTNAME -e ADVERTISE_IP -e JOIN --net=host --pid=host --privileged --cap-add SYS_ADMIN --device /dev/fuse -v /var/lib/storageos:/var/lib/storageos:rshared -v /run/docker/plugins:/run/docker/plugins -v /sys:/sys storageos/node:1.5.3 server
+ExecStartPre=/usr/bin/docker pull storageos/node:v2.0.0
```

Comment:
We should not change this. We don't  support this type of installation for v2

Reply:
Yeap fair point.